src/System.IO/tests/Stream/Stream.TestLeaveOpen.cs

```diff
@@ -22,37 +22,20 @@ public void StreamReaderTest()
             ms.Position = 0;
             Assert.True(ms.CanRead, "ERROR: Before testing, MemoryStream's CanRead property was false!  What?");
 
-            // Test leaveOpen with calling Dispose
+            // Test leaveOpen.
             StreamReader sr = new StreamReader(ms, System.Text.Encoding.UTF8, true, 1000, true);
             sr.Dispose();
-            Assert.True(ms.CanRead, "ERROR: After disposing a StreamReader with leaveOpen bool set, MemoryStream's CanRead property was false!");
```

Comment:
The lines I removed where added in #10038 to existing tests, and made them not compatible with netcoreapp1.0 any longer. Given that we should write tests for BinaryWriter.Close() anyway, I thought it would be best to remove this for now to have the old tests compiling against the previous surface area. I can instead add some #ifdefs surrounding this code and have the tests run twice (ncapp1.0 and ncapp1.1), do you think that's valuable?